.travis.yml

```diff
@@ -12,7 +12,6 @@ env:
 stages:
   - check
   - unit
-  - e2e
```

Comment:
This means that tests are not passing?

Reply:
e2e tests for Pravega are not applicable as BK e2e tests, as the objective there is to test Pravega functionality.
We would have to write a completely new e2e test suite for BK if we want e2e tests.

Reply:
> e expert eyes on it may be good. Consider adding them as reviewers too

Added @Tristan1900